lib/websocket.js

```diff
@@ -463,9 +463,17 @@ function initAsClient (address, protocols, options) {
   }
 
   this._isServer = false;
-  this.url = address;
 
-  const serverUrl = url.parse(address);
+  let serverUrl;
```

Comment:
We are still using `var` when `const` can not be used.

Reply:
FML, I've been writing to much ES6.